conf/app.routes

```diff
@@ -37,4 +37,10 @@ POST       /class-of-beneficiaries/when-added                               cont
 GET        /class-of-beneficiaries/check-details                            controllers.classofbeneficiary.add.CheckDetailsController.onPageLoad()
 POST       /class-of-beneficiaries/check-details                            controllers.classofbeneficiary.add.CheckDetailsController.onSubmit()
 
-GET        /:utr                                                            controllers.IndexController.onPageLoad(utr: String)
\ No newline at end of file
+GET        /:utr                                                            controllers.IndexController.onPageLoad(utr: String)
+
+GET        /class-of-beneficiaries/:index/remove                            controllers.classofbeneficiary.remove.RemoveClassOfBeneficiaryController.onPageLoad(index : Int)
```

Comment:
Don't these endpoints need to be defined before the /:utr? Otherwise the regex will capture the whole reverse route?

Reply:
I don't think they need to be, but I'm not certain so I've moved them up just in case.